docs/getting_started/docker/1.md

```diff
@@ -159,7 +163,7 @@ Let's use `qlooctl` to register some resolvers.
 
 ```bash
 # register findPetById for Query.pets (specifying no arguments)
-qlooctl resolvermap register -u petstore -f findPetById Query pets
+qlooctl resolvermap register -u petstore -f findPets Query pets
```

Comment:
Wasn't sure if this was intended to be `findPetById`. If it was, I'll revert it.